README.md

```diff
@@ -61,23 +61,29 @@ ReactJS based Presentation Library
 
 ## Getting Started
 
-The new best way to get started is by running `create-react-app my-presentation --scripts-version spectacle-scripts`. This will use `create-react-app` to create almost everything you need. This however, doesn't include publish scripts, and ejecting is required for fancy stuff.
+There are three ways to get started on your presentation. The first way is to run the following commend in your terminal.
 
-The second best way to get started is by using the [Spectacle Boilerplate](https://github.com/FormidableLabs/spectacle-boilerplate).
+`npx create-react-app my-presentation --scripts-version spectacle-scripts`
 
-Alternatively, you can `npm install spectacle` and write your own build configurations. We also provide full UMD builds (with a `Spectacle` global variable) of the library at `dist/spectacle.js` and `dist/spectacle.min.js` for more general use cases. You could, for example, include the library via a script tag with: `https://unpkg.com/spectacle@VERSION/dist/spectacle.min.js`.
+or
 
-Note that we have webpack externals for `react`, `react-dom`, and `prop-types`, so you will need to provide them in your upstream build or something like linking in via `script` tags in your HTML page for all three libraries. This comports with our project dependencies which place these three libraries in `peerDependencies`.
+`yarn create react-app my-presentation --scripts-version spectacle-scripts`
```

Comment:
I think we actually need to get rid of the yarn one - it should only be the npx one after re-reviewing the react docs

Reply:
@ebrillhart is it worth mentioning in the `Development` section that the corresponding yarn commend will also work?


## Development

After downloading the boilerplate, run the following commands on the project's root directory...

- `npm install`
- `rm -R .git` to remove the existing version control
- `npm start` to start up the local server or visit [http://localhost:3000/#/](http://localhost:3000/#/)

```
... and we are ready to roll. Note that the corresponding `yarn` commends  also work.
```
or

```
- `npm install` (you can also us `yarn`)
```